### DISCUSSION TITLE
The database shows garbled characters

### DISCUSSION THREAD

**Discussion by zhangsj** [dani-garcia/vaultwarden#2266] [Q&A]
use docker-compose.
mariadb

select * from users.

![image](https://user-images.githubusercontent.com/7639036/151514307-a63b06cc-31ca-4efc-bf24-0a8f959f17ba.png)

export csv garbled characters too.

**Comment by BlackDex** [comment]
Those fields are binary blobs, use `--binary-as-hex` to have it shown as hex instead of the actual blob-bytes.
And for mysqldump you could use `--hex-blob`.

**Comment by zhangsj** [comment]
csv Chinese display garbled characters。how fix

**Comment by BlackDex** [comment]
If you are referring to those 'garbled' strings in your screenshot, that is normal. Those are binary blobs. I'm not sure how you extract it into an csv but you would need to have that exported as hex as i mentioned in my previous post.

**Comment by zhangsj** [comment]
username is chinese .export csv garbled characters

![image](https://user-images.githubusercontent.com/7639036/151528878-83842c0d-f37f-4623-88b5-9a0023ba05ea.png)

**Reply by BlackDex** [reply]
That looks like an organization export from the web-vault.
That is not something Vaultwarden can fix since those are fully generated by the web-vault it self.
So this probably looks like something is wrong during the export, in that case you should check https://github.com/bitwarden/web/issues
